app/scripts/metamask-controller.js

```diff
@@ -198,6 +199,16 @@ module.exports = class MetamaskController extends EventEmitter {
 
     this.addressBookController = new AddressBookController(undefined, initState.AddressBookController)
 
+    this.threeBoxController = new ThreeBoxController({
```

Comment:
I believe this should be behind the feature flag as well, as `init` is called within the controller constructor.

Or maybe the flag should be used _within_ the controller... not sure.

Reply:
Did it in the controller here a203d5862
If we did it in the metamask controller, we would have to had some new approach for dynamically instantiating the threebox controller based on a state change.

Reply:
Right, good point. In the controller makes much more sense.